### ISSUE TITLE
[FEATURE REQUEST, RR2] Simplify RR2 error messages

### ISSUE THREAD

**Issue by SerafimArts** [spiral/roadrunner#514]
> Probably regression bug

## Problem

> Note: the code below has identical error

- RoadRunner 1.x contains concise and clear error messages:
```
WARN[0000] config: Config File ".rr" Not Found in "[path/to/directory]"
```

- RoadRunner 2.x сontains several tens of trace lines from which it is not clear what happened.
```
2021-01-30T20:08:14.894+0300    ERROR   container/init.go:74    error calling internal_init     {"vertex id": "config.Viper", "error": "open path/to/directory/.rr.yaml: The system cannot find the file specified."}
github.com/spiral/endure/pkg/container.(*Endure).callInitFn
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/init.go:74
github.com/spiral/endure/pkg/container.(*Endure).internalInit
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/init.go:26
github.com/spiral/endure/pkg/container.(*Endure).Initialize
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:379
reflect.Value.call
        reflect/value.go:476
reflect.Value.Call
        reflect/value.go:337
github.com/spiral/endure/pkg/fsm.(*FSMImpl).Transition
        github.com/spiral/endure@v1.0.0-beta21/pkg/fsm/fsm.go:180
github.com/spiral/endure/pkg/container.(*Endure).Init
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:312
github.com/spiral/roadrunner/v2/cmd/cli.handler
        github.com/spiral/roadrunner/v2/cmd/cli/serve.go:29
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.1.1/command.go:850
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.1.1/command.go:958
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spiral/roadrunner/v2/cmd/cli.Execute
        github.com/spiral/roadrunner/v2/cmd/cli/root.go:41
main.main
        command-line-arguments/main.go:68
runtime.main
        runtime/proc.go:204
2021-01-30T20:08:14.933+0300    ERROR   container/init.go:28    error occurred during the call INIT function    {"vertex id": "config.Viper", "error": "endure_call_i
nit_fn: Function call error: open path/to/directory/.rr.yaml: The system cannot find the file specified."}
github.com/spiral/endure/pkg/container.(*Endure).internalInit
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/init.go:28
github.com/spiral/endure/pkg/container.(*Endure).Initialize
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:379
reflect.Value.call
        reflect/value.go:476
reflect.Value.Call
        reflect/value.go:337
github.com/spiral/endure/pkg/fsm.(*FSMImpl).Transition
        github.com/spiral/endure@v1.0.0-beta21/pkg/fsm/fsm.go:180
github.com/spiral/endure/pkg/container.(*Endure).Init
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:312
github.com/spiral/roadrunner/v2/cmd/cli.handler
        github.com/spiral/roadrunner/v2/cmd/cli/serve.go:29
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.1.1/command.go:850
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.1.1/command.go:958
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spiral/roadrunner/v2/cmd/cli.Execute
        github.com/spiral/roadrunner/v2/cmd/cli/root.go:41
main.main
        command-line-arguments/main.go:68
runtime.main
        runtime/proc.go:204
2021-01-30T20:08:14.946+0300    ERROR   container/endure.go:382 error during the internal_init  {"error": "endure_internal_init: Function call error:\n\tendure_call_
init_fn: open path/to/directory/.rr.yaml: The system cannot find the file specified."}
github.com/spiral/endure/pkg/container.(*Endure).Initialize
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:382
reflect.Value.call
        reflect/value.go:476
reflect.Value.Call
        reflect/value.go:337
github.com/spiral/endure/pkg/fsm.(*FSMImpl).Transition
        github.com/spiral/endure@v1.0.0-beta21/pkg/fsm/fsm.go:180
github.com/spiral/endure/pkg/container.(*Endure).Init
        github.com/spiral/endure@v1.0.0-beta21/pkg/container/endure.go:312
github.com/spiral/roadrunner/v2/cmd/cli.handler
        github.com/spiral/roadrunner/v2/cmd/cli/serve.go:29
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.1.1/command.go:850
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.1.1/command.go:958
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.1.1/command.go:895
github.com/spiral/roadrunner/v2/cmd/cli.Execute
        github.com/spiral/roadrunner/v2/cmd/cli/root.go:41
main.main
        command-line-arguments/main.go:68
runtime.main
        runtime/proc.go:204
2021/01/30 20:08:14 handle_serve_command: Init error:
        endure_initialize:
        endure_internal_init: Function call error:
        endure_call_init_fn: open path/to/directory/.rr.yaml: The system cannot find the file specified.

```

## Solution

I propose to simplify error messages, or highlight the message directly, so that it would be clear what exactly happened.

**Comment by 48d90782** [comment]
Thanks for sharing your thoughts. That can be easily tuned by setting endure log level to panic. We haven't decided yet what the error messages will be. This option will most likely be set through the configuration.
```
2021/01/30 23:40:19 handle_serve_command: Init error:
	endure_initialize:
	endure_internal_init: Function call error:
	endure_call_init_fn: open /cmd/.rrr.yaml: no such file or directory
```

**Comment by wolfy-j** [comment]
Let's discuss this on Monday.

**Comment by 48d90782** [comment]
Fixed in the `2.0.4` release:
![image](https://user-images.githubusercontent.com/8040338/114544971-09cca000-9c64-11eb-9bf6-20dc1784fc9a.png)
By default, endure log level now is `error` which is not as verbose as shown before.
